openwisp_users/apps.py

```diff
@@ -98,11 +98,18 @@ def set_default_settings(self):
     def connect_receivers(self):
         OrganizationUser = load_model('openwisp_users', 'OrganizationUser')
         OrganizationOwner = load_model('openwisp_users', 'OrganizationOwner')
+        Organization = load_model("openwisp_users" , "Organization") 
```

Comment:
We use single quotes (`'`), please maintain consistency, applies to all other changes.